app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/util/CipherViewExtensions.kt

```diff
@@ -48,6 +48,7 @@ fun CipherView.toViewState(
                     password = login?.password.orEmpty(),
                     totp = login?.totp,
                     canViewPassword = this.viewPassword,
+                    canEdit = this.edit,
```

Comment:
This could just be:

```kotlin
canRemovePasskey = this.edit && this.viewPassword,
```

This way the screen never needs to know the extra complexity of defining this boolean.